client/src/components/Navbar/Navbar.tsx

```diff
@@ -32,63 +41,82 @@ const Navbar: React.FC = () => {
     handleClose();
     logout();
   };
-
+  console.log(loggedInUser);
   return (
-    <AppBar position="static">
-      <Toolbar>
-        <IconButton size="large" edge="start" color="inherit" aria-label="menu" sx={{ mr: 2 }}>
-          <MenuIcon />
-        </IconButton>
-        <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
-          My App
-        </Typography>
+    <AppBar className={classes.root} position="static">
+      <Toolbar className={classes.toolbar}>
+        <Link to={'/home'}>
+          <img src={logo} alt="Pet Sitter Logo"></img>
+        </Link>
+        {!loggedInUser && (
+          <>
+            <Box className={classes.flexContainer}>
+              <Link className={classes.link} to={'/'}>
+                BECOME A SITTER
```

Comment:
Let's avoid the usage of CAPITAL CASE text for accessibility reasons (some screen readers interpret them as abbreviations). We can use a CSS class to transform text instead 

Ditto where applicable